### ISSUE TITLE
Min zoom

### ISSUE THREAD

**Issue by newiep01** [anvaka/panzoom#115]
Hi, is it possible to detect the zoom has reached the min zoom?

**Comment by rbonomo** [comment]
Yeah, look at the matrix transform style property applied to your controller element.
